### PR TITLE
Some "fixes"

### PR DESCRIPTION
**Pull request by astagi**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -133,11 +134,15 @@ nav {
 }
 
 @media (max-width:500px) {
+    main {
+        text-align: justify;
```
Comment (patrick91) [inline css/main.css]: I don't usually like justified text. Can we remove it? :)

**Review by patrick91 — CHANGES_REQUESTED** [review]

File: css/main.css (hunk 1 above)
Comment (astagi) [reply css/main.css]: Sure... Which value do you suggest to use instead?

Comment (patrick91) [reply css/main.css]: center looks fine!

Comment (astagi) [reply css/main.css]: Done :)

**Pull request merged**